selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/factory/AuthenticateAndAcceptFactoryThroughGitHubOAuthTest.java

```diff
@@ -107,4 +111,43 @@ public void loginThroughGitHubOAuthAndAcceptFactory() throws Exception {
     projectExplorer.waitProjectExplorer();
     notificationsPopupPanel.waitExpectedMessageOnProgressPanelAndClosed("Project Spring imported");
   }
+
+  private void deleteAllWorkspaces(List<String> workspaces) {
```

Comment:
This method is about action inside Workspaces page of dashboard and so shall be places in the appropriate page object.

Reply:
By the way - there is a check-box in left top of _Workspaces page_ which allows to select all workspaces and delete all of them at once, instead of removing they one by one:
![screenshot-2017-10-20 codenvy workspaces](https://user-images.githubusercontent.com/1197777/31819810-0c8dc848-b5a7-11e7-8d2d-655da1ac6b91.png)

Reply:
Yes its more simply, but we are not have methods for working with this check-box

Reply:
It a chance to add it :-)

Reply:
But this is not a goal of this issue, I used existed methods :-)